House-of-glass/src/Contexts/Cart/CartContext.jsx

```diff
@@ -0,0 +1,169 @@
+import { createContext, useContext, useReducer, useEffect } from "react";
+import axios from "axios";
+import { cartReducer } from "./CartReducer";
+import Toast from "components/Toast/Toast";
+
+const CartContext = createContext();
+
+const useCart = () => useContext(CartContext);
+
+const CartProvider = ({ children }) => {
+  const initialCartState = { cart: [], cartCount: 0, cartSum: 0 };
+  const [cartState, cartDispatch] = useReducer(cartReducer, initialCartState);
+
+  const token = localStorage.getItem("Token");
+  const header = {
+    headers: {
+      authorization: token,
+    },
+  };
```

Comment:
This is a good approach of representing the headers parameter that we often pass while making API calls

Reply:
Thank You 🙃